proto/proto/penumbra/core/ibc/v1alpha1/ibc.proto

```diff
@@ -78,6 +78,17 @@ message Ics20Withdrawal {
   string source_channel = 9;
 }
 
+message Ics20WithdrawalPlan {
+  string destination_chain_id = 1;
+  string destination_chain_address = 2;
+  crypto.v1alpha1.Denom denom = 3;
+  crypto.v1alpha1.Amount amount = 4;
+  string source_channel = 5;
+  uint64 timeout_height = 6;
+  uint64 timeout_timestamp = 7;
+  crypto.v1alpha1.Address return_address = 8;
+}
```

Comment:
Does this differ from the `Ics20Withdrawal` action? If there's no blinding factors or other encrypted data etc., we can just use the action as a plan of itself, similar to the other transparent actions.